packages/neon_framework/packages/account_repository/pubspec.yaml

```diff
@@ -5,6 +5,7 @@ publish_to: none
 
 environment:
   sdk: ^3.0.0
+  flutter: ^3.22.0
```

Comment:
Intentional and necessary?
I thought repositories should be pure Dart.

Reply:
the repo depends on `neon_framework` for the storages. This pulls in `intl` which requires flutter.
I'm working on a separate storage layer that does not pull this in.

Reply:
In this PR or separately?

Reply:
Sparate one.
CI still fails to bootstrap :(